tests/installation/releasenotes.pm

```diff
@@ -21,7 +21,9 @@ sub run(){
         }
     }
     send_key 'alt-o';   # exit release notes window
-    send_key 'alt-e';   # select region as previous selected
+    if (!get_var("UPGRADE")) {
```

Comment:
Well it's useful also in graphical mode , but test in graphical mode will pass with or without alt-e as the difference is minimal :)
https://openqa.suse.de/tests/97127/modules/installer_timezone/steps/1